pkgs/tools/misc/wakatime/default.nix

```diff
@@ -1,7 +1,8 @@
 { stdenv, buildPythonApplication, fetchFromGitHub }:
 
 buildPythonApplication rec {
-  name = "wakatime-${version}";
+  baseName = "wakatime";
+  name = "${baseName}-${version}";
```

Comment:
name can be dropped

Reply:
Does it default to `${pname}-${version}`?

Reply:
In ` buildPython*`, yes.